Tools/LyTestTools/ly_test_tools/o3de/settings.py

```diff
@@ -169,7 +170,7 @@ def get_key(self, key_path, default_value=None):
                 obj = obj[token]
             return obj
         except KeyError as err:
-            logger.error(f'KeyError when retrieving {key_path} from file {self._file_path}: {err}')
+            logger.error(f'KeyError originating from LyTT when retrieving {key_path} from file {self._file_path}: {err}')
```

Comment:
I don't think we can state this, as this 'settings' module was written to interact with non-LyTT settings files. Though it may not currently be used in any tests.